hermes-console/static/js/console/topic/TopicController.js

```diff
@@ -47,6 +47,14 @@ topics.controller('TopicController', ['TOPIC_CONFIG', 'TopicRepository', 'TopicM
             $scope.preview = preview;
         });
 
+        $scope.humanReadableSize = function (size) {
```

Comment:
this should be angular filter, not scope function, so we could use `topic.maxSize | humanReadableSize` syntax in all places of app